Add more details to README, Update Seeds

leahriffell opened the pull request:
### Code Highlights:
 * Added some more details to the README. Not a polished version but this will provide more of a template when we're ready to start dropping in final deets. I wasn't sure what to put in the general description (what the app is in general) so please leave any ideas you have!
 * Changes Seeds so users have Date instead of DateTime.
 * Changes Sees so that the upon re-seed, the ID numbers are reset

### Where should the reviewer start?
 * README

### Have Tests Been Added?
- [x] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.

### Issues:
* Addresses #70

### Tracking Consistency:
- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] looked at PR preview to check spelling, syntax, formatting, and completion
- [x] Rubocop Violations:

leahriffell committed:
Add more details to README

LHJE committed:
Merge branch 'main' of https://github.com/Turing-Eras/api into more-readme

LHJE committed:
Small update to Readme

LHJE committed:
Merge branch 'delete_event' of https://github.com/Turing-Eras/api into more-readme

LHJE committed:
Update README with schema picture and test coverage

AngelaGuardia commented on README.md:
```diff
@@ -1,27 +1,70 @@
-[description]
-[table of contents]
-[content ideas: link to deployed app, setup, test suite, GQL queries and mutations, database schema and description of each table, contributors]
+This GraphQL on Rails API [insert live endpoint link] serves queries and mutations to Eras [insert live FE link], an application that allows you to visualize your life wholistically in order to live more intentionally.
+
+By providing info on personal events, you will be shown a visual calendar of your life. The intent is that by viewing the seemingly fleeting weeks that add up to years and decades, you are able to cherish the little moments and appreciate [insert stuff].
+
```
This sounds pretty good Leah! Thanks for doing this. 
```suggestion
This GraphQL on Rails API [insert live endpoint link] serves queries and mutations to Eras [insert live FE link], an application that allows you to visualize your life wholistically in order to live more intentionally.

By providing info on personal events, you will be shown a visual calendar of your life. The intent is that by viewing the seemingly fleeting weeks that add up to years and decades, you are able to appreciate the little moments, cherish the time you have with loved ones, and live with more intention.

```

LHJE commented on README.md:
```diff
@@ -1,27 +1,70 @@
+- Angela Guardia |  [Github](https://github.com/AngelaGuardia)  |  [LinkedIn](https://www.linkedin.com/in/angela-guardia/)
+- Daniel Halverson |  [Github](https:Gi.com/dhalverson)  |  [LinkedIn](https://www.linkedin.com/in/daniel-halverson/)
+- Leah Riffell |  [Github](https://github.com/leahriffell)  |  [LinkedIn](https://www.linkedin.com/in/leah-riffell/)
+- Luke Hunter James-Erickson  |  [Github](https://github.com/LHJE)  |  [LinkedIn](https://www.linkedin.com/in/luke-hunter-james-erickson-b65682143/)
```
All of this looks so good!  Thank you!

dhalverson replied:
Yes thank you!

LHJE reviewed (approved):
Looks great Leah!  We can of course keep adding more in as we go.

LHJE committed:
Update README with added mutations and queries

LHJE committed:
Adding Angela's comments back in after they disappeared.

Co-Authored-By: Angela Guardia <AngelaGuardia@users.noreply.github.com>

LHJE committed:
Merge branch 'main' of https://github.com/Turing-Eras/api into more-readme

LHJE committed:
Attempt to fix formatting error on README Queries section

LHJE committed:
Attempt to fix formatting error on README Queries section

LHJE committed:
Merge branch 'more-readme' of https://github.com/Turing-Eras/api into more-readme

LHJE committed:
Merge branch 'main' of https://github.com/Turing-Eras/api into more-readme

LHJE committed:
Update update_event_spec to expect a Date

LHJE committed:
Update README with attempts to fix formatting issue

LHJE committed:
More attempts at getting README formatting working

LHJE committed:
Update README with more formatting

LHJE committed:
Update README attempting to fix formatting issues

LHJE committed:
Add another attempt at fixing formatting issue

LHJE committed:
Add another attempt to fix README formatting issues

LHJE committed:
Add another attempt to fix README formatting issues

LHJE committed:
Merge branch 'more-readme' of https://github.com/Turing-Eras/api into more-readme

LHJE committed:
Update README to fix formatting issues...
... It's not as pretty as as using the `<summary>Example Text</summary> 
thing with the titles, but that was not working

LHJE commented on README.md:
````diff
@@ -1,27 +1,232 @@
+## GraphQL Schema
+- Queries available:
+  - Get Single Era: <details>
+    ```
````
The Prettier way of doing this would be to do on line 57:
```
 - <details><summary>Get Single Era:</summary>
```
But for some reason the notes between the ` ``` ` aren't being formatted in the correct way.  This is the compromise.

LHJE reviewed (approved):
Looking Good!

LHJE committed:
Update README with create Era mutation

LHJE committed:
Update README with create Era mutation

Co-Authored-By: Angela Guardia <AngelaGuardia@users.noreply.github.com>

LHJE committed:
Merge branch 'more-readme' of https://github.com/Turing-Eras/api into more-readme

LHJE committed:
Update Seeds so users have Date instead of DateTime

LHJE committed:
Update README with updateEra mutation

LHJE committed:
Update seeds to destroy the old ones on re-seed

LHJE committed:
Merge branch 'main' into more-readme

dhalverson reviewed (approved):
Reviewed and approved

LHJE committed:
Update Gems with PK Sequence

LHJE committed:
Merge branch 'more-readme' of https://github.com/Turing-Eras/api into more-readme

LHJE committed:
Update Gemfile with PK Sequence

LHJE committed:
Revise README mutation for clarity
